examples/powertools-examples-core/serverless/serverless.yml

```diff
@@ -0,0 +1,117 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: hello
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: java11
```

Comment:
In the pom it's 1.8...

Reply:
That's the same for all our example projects. Java 8 at build and Java11 at runtime:
https://github.com/aws-powertools/powertools-lambda-java/blob/ef11a64df17ee1cbf736aacbd308c9f191d21a8c/examples/powertools-examples-core/sam/template.yaml#L10

Reply:
yes :(